internal/test/test_test.go

```diff
@@ -124,6 +124,21 @@ func TestExecutingTests(t *testing.T) {
 		assert.NoError(t, results[script.Filename][0].Error)
 	})
 
+	t.Run("with helper script import", func(t *testing.T) {
+		t.Parallel()
+		_, state, _ := util.TestMocks(t)
+
+		// Execute script
+		script := tests.TestScriptWithHelperImport
+		testFiles := make(map[string][]byte, 0)
+		testFiles[script.Filename] = script.Source
```

Comment:
```suggestion
		testFiles := map[string][]byte{
			script.Filename: script.Source,
		}
```

Reply:
Fixed in https://github.com/onflow/flow-cli/pull/1030/commits/6c6d17453e5a5167fa7846e590bede5a9b78fdfa